documentation/asciidoc/topics/con_creating_caches.adoc

```diff
@@ -2,7 +2,7 @@
 = Creating {brandname} Caches
 To create caches when running {brandname} on {k8s}, you should:
 
-* Use `Cache` CR as the primary mechanism for creating caches.
+* Use `Cache` CR as the mechanism for creating caches through the {openshiftshort} front end. 
```

Comment:
@Crumby This is the same wording as on the "master" branch. Changing from saying "primary" mechanism to something more generic. Sound OK to you?